### PR TITLE
Adding documentation for theme system

### PR DESCRIPTION
**Pull request by BrooklynKing**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -41,21 +41,28 @@ The video player supports the following video formats: `MP4`, `WebM`, `HLS`, `DA
...
     },
     src: 'http://my-url/video.mp4',
     preload: 'metadata'
-  });
+  };
+  const theme = {
+    progressColor: "#AAA"
+  }
+
+  const player = Playable.create(config, theme);
```
Comment (bodia-uz) [inline docs/index.md]: I think the first example in `index.md` should be minimal code to run player.

Mb remove theme config here and change text "You can pass your own [theme](/themes)" with smth like "You can change player theme with your config" + examle

Comment (kupriyanenko) [reply docs/index.md]: agree

File: src/default-modules/ui/core/theme/theme-service.ts [hunk 2]
```diff
@@ -31,12 +31,17 @@ class ThemeService implements IThemeService {
...
+    // setTimeout here is for calling `attach` after all modules resolved.
     setTimeout(() => {
       this._styleSheet && this._styleSheet.attach();
     }, 0);
   }
 
+  /**
+   * Method for setting theme for player instance
+   *
+   * @param themeConfig - You can check available theme parameters in "Themes" section
```
Comment (kupriyanenko) [inline src/default-modules/ui/core/theme/theme-service.ts]: try to use links here.

Comment (kupriyanenko) [reply src/default-modules/ui/core/theme/theme-service.ts]: and add simple example

**Pull request merged**